src/main/java/edu/hw1/Task5.java

```diff
@@ -0,0 +1,45 @@
+package edu.hw1;
+
+@SuppressWarnings("MagicNumber")
+public final class Task5 {
+    private Task5() {
+    }
+
+    public static boolean isPalindromeDescendant(int num) throws NumberFormatException {
+        if (num < 0) {
+            throw new NumberFormatException("Only positive numbers are allowed");
+        }
+        String numStr = String.valueOf(num);
+        if (numStr.length() % 2 != 0) {
+            throw new NumberFormatException("Only even amount of digits is allowed");
+        }
+        int numInt = Integer.parseInt(numStr);
+
+        int reversedNum = reverseNum(numInt);
+
+        if (Integer.parseInt(numStr) == reversedNum) {
+            return true;
+        } else {
```

Comment:
`Enhancement`, после return можно не ставить else 
```suggestion
        } 
```

Reply:
Это уменьшит вложенность